resources/linux/flatpak/manifest.json.in

```diff
@@ -0,0 +1,41 @@
+{
+  "id": "<%= appId %>",
+  "base": "<%= baseAppId %>",
+  "base-version": "<%= baseAppVersion %>",
+  "runtime": "<%= runtimeId %>",
+  "sdk": "<%= sdkId %>",
+  "runtime-version": "<%= runtimeVersion %>",
+  "command": "<%= atomExecutableName %>",
+  "finish-args": [
+    /* Allow access to developer tools */
+    "--allow=devel",
+    "--talk-name=org.freedesktop.Flatpak",
+    /* X11 + XShm access */
+    "--share=ipc", "--socket=x11",
+    /* Open GL */
+    "--device=dri",
+    /* Sound access */
+    "--socket=pulseaudio",
+    /* We want full fs access */
+    "--filesystem=host",
+    /* Chromium uses a socket in tmp for its singleton check */
+    "--filesystem=/tmp",
+    /* Needs to talk to the network: */
+    "--share=network",
+    /* Notifications with libnotify */
+    "--talk-name=org.freedesktop.Notifications"
+  ],
+  "branch": "<%= channel %>",
+  "modules": [
+    {
+      "name": "xkbfile",
```

Comment:
When I first started this port I didn't need this. But looks like this is needed from a recent bump in `atom-keymap`.

From what I can tell this is a specific atom requirement and not a general electron requirement, so I'm just building it into the app directly.